content/2020/03/04/csswg.rst

```diff
@@ -0,0 +1,53 @@
+public: yes
+author: miriam
+tags: [Code, CSS]
+headline:
+  - tagline: Helping define the future of styles on the web
+image:
+  - src: 'projects/w3c.jpg'
+quotes:
+  - text: |
+      I think it's a brilliant idea.
+      Almost want to stop talking about whether or not to do it,
+      and just start talking syntax.
+    name: 'Florian Rivoal'
+    role: 'W3C Invited Expert'
+    slug: 'on-demand'
+summary: |
+  **I've joined the CSS Working Group
+  as a W3C Invited Expert**
+  to help to develop the next level of
+  Cascading & Inheritance,
+  in addition to other CSS standards.
+  I'm also currently active in the
+  `CSS4 <https://www.w3.org/community/css4/>`_
+  and `Design Token <https://www.w3.org/community/design-tokens/>`_
+  Community Groups.
+
+
+W3C Invited Expert on the CSS Working Group
+===========================================
+
+I was invited to join
+after the CSSWG approved work on my
+proposal for extending the
+CSS Cascade & Inheritance module,
+so that authors can define
+`custom CSS origins`_.
+Jen Simmons presented the proposal
+at a CSSWG "face-to-face" meeting,
+along with `supporting slides`_,
+and I'll write a more detailed article
+with details about the proposal soon.
```

Comment:
Not a dealbreaker, but what if this last part is a separate sentence? Like:

Jen Simmons presented the proposal at a CSSWG "face-to-face" meeting, along with `supporting slides`_. Stay tuned for a detailed article about the proposal.

Reply:
pushed the update